src/main.rs

```diff
@@ -101,20 +103,20 @@ pub fn batched_insertion(tree: &mut Smt) -> Result<(), MerkleError> {
     let mutations = tree.compute_mutations(new_pairs);
     let compute_elapsed = now.elapsed().as_secs_f64() * 1000_f64; // time in ms
 
-    let now = Instant::now();
-    tree.apply_mutations(mutations)?;
-    let apply_elapsed = now.elapsed().as_secs_f64() * 1000_f64; // time in ms
-
     println!(
-        "An average insert-batch computation time measured by a {NUM_INSERTIONS}-batch into an SMT with {size} leaves over {:.1} ms is {:.0} μs",
+        "An average insert-batch computation time measured by a 1k-batch into an SMT with {size} leaves over {:.1} ms is {:.0} μs",
```

Comment:
```suggestion
        "The average insert-batch computation time measured by a {insertions}-batch into an SMT with {size} leaves over {:.1} ms is {:.0} μs",
```

Is the batch here not of size `insertions` or is this referring to something else?